<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RS232</class>
 <widget class="QMainWindow" name="RS232">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RS232</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_SendMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_Port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Connect">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>連線</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Disconnect">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>斷開</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SendMessage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>傳送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rtb_ReceiveMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>451</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
